src/components/biggive-tabbed-content/biggive-tabbed-content.tsx

```diff
@@ -1,24 +1,122 @@
-import { Component, Prop, h } from '@stencil/core';
+import { Component, Prop, Element, h } from '@stencil/core';
+import { brandColour } from '../../globals/brand-colour';
+import { spacingOption } from '../../globals/spacing-option';
 
 @Component({
   tag: 'biggive-tabbed-content',
   styleUrl: 'biggive-tabbed-content.scss',
   shadow: true,
 })
 export class BiggiveTabbedContent {
-  /**
-   * Space below component
-   */
-  @Prop() spaceBelow: number = 0;
+  @Element() host: HTMLBiggiveTabbedContentElement;
+
+  @Prop() spaceBelow: spacingOption = 0;
+
+  @Prop() textColour: brandColour = 'black';
+
+  @Prop() selectedTextColour: brandColour = 'primary';
+
+  @Prop() navigationHighlightColour: brandColour = 'grey-medium';
+
+  @Prop() selectedNavigationHighlightColour: brandColour = 'primary';
+
+  @Prop() buttonBackgroundColour: brandColour = 'white';
+
+  @Prop() buttonIconColour: brandColour = 'primary';
+
+  currentTab = 0;
+  children: Array<HTMLBiggiveTabElement> = [];
+
+  componentWillLoad() {
+    this.children = Array.from(this.host.children) as Array<HTMLBiggiveTabElement>;
+  }
 
-  /**
-   * Default text colour
+  componentDidRender() {
+    this.showTab(0);
+  }
+
+  /*
+   * Shows the i'th tab, counting from zero. Does nothing if i out of range.
    */
-  @Prop() defaultTextColour: string = 'black';
+  showTab(i: number) {
+    const tabs = this.host.shadowRoot?.querySelectorAll('.navigation ul li')!;
+
+    if (tabs && i >= 0 && i <= tabs?.length - 1) {
```

Comment:
The `tabs &&` part is redundant here isn't now that we've declared that tabs is always non-null

```suggestion
    if (i >= 0 && i <= tabs?.length - 1) {
```

Reply:
Updated as per your suggestion